portfolio/src/main/java/com/google/sps/servlets/GlobalWarmingDataServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns global warming sentiment data as a JSON object */
+@WebServlet("/global-warming-sentiment-data")
+public class GlobalWarmingDataServlet extends HttpServlet {
+
+  private LinkedHashMap<String, Integer> globalWarmingSentiment = new LinkedHashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/global-warming-sentiment-data.csv"));
+    
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
+
+      String tweet = (String) cells[0];
+      Integer existence = 0;
```

Comment:
For existence, leave a comment similar to 

// 0 represents that person believes client change exist, else 1 (or make it whatever why you want)